src/routes/write-exercise/__tests__/WriteExerciseScreen.spec.tsx

```diff
@@ -0,0 +1,78 @@
+import { mockUseLoadFromEndpointWitData } from '../../../testing/mockUseLoadFromEndpoint'
+import React from 'react'
+import { fireEvent, render } from '@testing-library/react-native'
+import WriteExerciseScreen from '../WriteExerciseScreen'
+import createNavigationMock from '../../../testing/createNavigationPropMock'
+import { RouteProp } from '@react-navigation/native'
+import { RoutesParamsType } from '../../../navigation/NavigationTypes'
+import { DocumentTypeFromServer } from '../../../hooks/useLoadDocuments'
+import labels from '../../../constants/labels.json'
+import { ReactTestInstance } from 'react-test-renderer'
+
+jest.mock('../../../components/AudioPlayer', () => {
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
```

Comment:
Maybe we could just disable this rule for tests, does not have to be done in this PR oc though.